### PR TITLE
[MIG][website_snippet_marginless_gallery] Migrate to v9

### PR DESCRIPTION
**Pull request by yajo**
- Relicense to LGPL.
- Move tour to demo asset.
- Remove precompiled CSS.
- Update snippet options JS API.
- Update tour's API and definition.

@Tecnativa

### CODE REVIEW

File: website_snippet_marginless_gallery/__openerp__.py [hunk 1]
```diff
@@ -2,14 +2,13 @@
 {
     "name": "Marginless Gallery Snippet",
     "summary": "Add a snippet to have a marginless image gallery",
-    "version": "8.0.1.0.1",
+    "version": "9.0.1.0.1",
```
Comment (pedrobaeza) [inline website_snippet_marginless_gallery/__openerp__.py]: 9.0.1.0.0

File: website_snippet_marginless_gallery/tests/__init__.py [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-# © 2015 Grupo ESOC Ingeniería de Servicios, S.L.U. - Jairo Llopis
-# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+# Copyright 2015-today Jairo Llopis <jairo.llopis@tecnativa.com>
```
Comment (pedrobaeza) [inline website_snippet_marginless_gallery/tests/__init__.py]: Use exact year, not today

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Tests are broken and I can't try on runbot due to `compass installation`

File: website_snippet_marginless_gallery/tests/__init__.py (hunk 2 above)
Comment (yajo) [reply website_snippet_marginless_gallery/tests/__init__.py]: I got inspired by https://github.com/facebook/react/commit/bef45b0b1a98ea9b472ba664d955a039cf2f8068 and the assumption that Facebook has good lawyers.

Comment (pedrobaeza) [reply website_snippet_marginless_gallery/tests/__init__.py]: We don't use this rule in OCA (which is abusing), so change it

**Comment by pedrobaeza** [comment]
I still can't try on runbot.

**Comment by pedrobaeza** [comment]
I have been trying on runbot but I'm not able to change the background image of any tile, as "Customize" doesn't appear when selecting a tile.

**Comment by yajo** [comment]
I guess it should be fixed now, let's try when runbot builds.

**Comment by pedrobaeza** [comment]
@Yajo, I'm still not able to customize tile background image.

**Review by pedrobaeza — APPROVED** [review]
Now is finally working.

@rafaelbn please review

**Review by rafaelbn — APPROVED** [review]
Super 👍 . Tested in runbot

**Comment by yajo** [comment]
Squashed for merge.

**Pull request merged**